### PR TITLE
MTM1M3TS implementation

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR implements the `MTM1M3TS` component.

### CODE REVIEW

File: love/src/components/MainTel/M1M3TS/Info/Info.jsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+      differentialTemperature: differentialTemperature[sensorIndex] ?? 0,
+      fanRPM: fanRPM[sensorIndex] ?? 0,
+      fanBreaker: fanBreaker[sensorIndex] ?? false,
+      heaterDisabled: heaterDisabled[sensorIndex] ?? false,
+      state: {
+        ilc: {
+          name: ilcState,
+          status: m1m3tsILCStateToStyle[ilcState],
+        },
+        // enabled: {
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/Info/Info.jsx:111]: Will this variable be used? If not, maybe it should be removed.

File: love/src/components/MainTel/M1M3TS/Info/Info.jsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+      differentialTemperature,
+      fanRPM,
+      fanBreaker,
+      heaterDisabled,
+    } = this.props;
+
+    const fcuIndex = M1M3TSFanCoilPositions.findIndex((fcu) => fcu.id === id);
+    const sensorIndex = sensorReferenceId.indexOf(fcuIndex);
+
+    // const enabledData = enabled[sensorIndex];
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/Info/Info.jsx:84]: Will those variables be used? If not, maybe they should be removed.

File: love/src/components/MainTel/M1M3TS/M1M3TS.jsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+            toggleFcuIDs={(show) => this.setState({ showFcuIDs: show })}
+            toggleTemperature={(show) => this.setState({ showDifferentialTemp: show })}
+            toggleWarnings={(show) => this.setState({ showWarnings: show })}
+          />
+        </div>
+
+        <div className={styles.selectorContainer}>
+          <Selector
+            sensorReferenceId={referenceId}
+            // enabled={enabledFCU}
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/M1M3TS.jsx:135]: Will this variable be used? If not, maybe it should be removed.

File: love/src/components/MainTel/M1M3TS/M1M3TS.jsx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+            selectedId={this.state.selectedSensor}
+            showDifferentialTemp={this.state.showDifferentialTemp}
+          />
+        </div>
+
+        <div className={styles.infoContainer}>
+          <Info
+            sensorReferenceId={referenceId}
+            ilcFCU={ilcFCU}
+            // enabled={enabledFCU}
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/M1M3TS.jsx:172]: The same as above

File: love/src/components/MainTel/M1M3TS/Mixing/Mixing.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  render() {
+    const {value, limit} = this.props;
+    return (
+      <>
+      <div className={styles.containerTop}>
+        <span className={styles.title}>Mixing Value</span>
+      </div>
+      <div>
+        <ProgressBar
+          /* targetValue={100} */
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/Mixing/Mixing.jsx:29]: The same as above

File: love/src/components/MainTel/M1M3TS/Mixing/Mixing.jsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        />
+      </div>
+      <div className={styles.containerBottom}>
+        <span className={styles.labelValue } style={{ left: 0 }}>
+          <span className={styles.label}>Valve Position </span>
+          <span className={styles.value}>{`${defaultNumberFormatter(value, 1)}%`}</span>
+        </span>
+        <span className={styles.labelValue } style={{ right: 0 }}>
+        {/*   
+          <span className={styles.label}>Measured </span>
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/Mixing/Mixing.jsx:42]: The same as above

File: love/src/components/MainTel/M1M3TS/Selector/Selector.jsx [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+        maxRadius = Math.floor(Math.sqrt(act.position[0] * act.position[0] + act.position[1] * act.position[1]));
+      }
+    });
+
+    const margin = 6;
+    xMin += margin;
+    xMax -= margin;
+    yMin -= margin;
+    yMax += margin;
+    // console.log(xMin, xMax, yMin, yMax, maxRadius);
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/Selector/Selector.jsx:188]: This console.log should be removed

File: love/src/components/MainTel/M1M3TS/Info/Info.jsx (hunk 1 above)
Comment (sebastian-aranda) [reply love/src/components/MainTel/M1M3TS/Info/Info.jsx:111]: Same as stated above.

File: love/src/components/MainTel/M1M3TS/M1M3TS.jsx (hunk 3 above)
Comment (sebastian-aranda) [reply love/src/components/MainTel/M1M3TS/M1M3TS.jsx:135]: Same as stated above.

File: love/src/components/MainTel/M1M3TS/Selector/Selector.jsx [hunk 8]
```diff
@@ -0,0 +1,413 @@
...
+        <circle
+          id="background-circle"
+          className={this.state.sensors.length > 0 ? styles.circleOverlay : styles.circleOverlayDisabled}
+          cx={this.state.width / 2}
+          cy={this.state.width / 2}
+          key={'background'}
+          r={this.state.width / 2}
+        />
+
+        {/* <circle
```
Comment (areyesd14) [inline love/src/components/MainTel/M1M3TS/Selector/Selector.jsx:326]: Will this element be used? If not, maybe it should be removed.

File: love/src/components/MainTel/M1M3TS/Mixing/Mixing.jsx (hunk 5 above)
Comment (sebastian-aranda) [reply love/src/components/MainTel/M1M3TS/Mixing/Mixing.jsx:29]: Will remove it on the next commit. Thanks!

**Review by areyesd14 — APPROVED** [review]
Everything looks good. I left only a few single comments.

File: love/src/components/MainTel/M1M3TS/Selector/Selector.jsx (hunk 8 above)
Comment (sebastian-aranda) [reply love/src/components/MainTel/M1M3TS/Selector/Selector.jsx]: You are correct, removed on the next commit. Thank!

**Pull request merged**